appveyor.yml

```diff
@@ -140,11 +145,6 @@
 
                 cd build
 
-                # Make sure we have Python27-64 before any other version; Python path per https://www.appveyor.com/docs/installed-software#python
```

Comment:
C++ build is failing now. Looks like we may need this in a common locations for 64-bit builds.
https://ci.appveyor.com/project/sapek/bond/build/1.0.1277/job/r229e9ypw9600t8b